### PR TITLE
Sign over all applicable square sizes by default

### PR DESCRIPTION
**Pull request by evan-forbes**
## Description

this could still use some cleaning up, but this PR changes things so that we sign over all applicable square sizes when creating a wirePFD

cc @renaynay 

closes: #239

### CODE REVIEW

**Review by SweeXordious — COMMENTED** [review]
Where is the whole signature happening ? It seems that this PR only moves functionality from a place to another?  I guess I am missing some change

**Review by renaynay — APPROVED** [review]
lgtm

File: x/payment/types/square_sizes.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package types
+
+import (
+	"github.com/tendermint/tendermint/pkg/consts"
+)
+
+// https://github.com/celestiaorg/celestia-app/issues/236
+// https://github.com/celestiaorg/celestia-app/issues/239
```
Comment (liamsi) [inline x/payment/types/square_sizes.go:8]: It's unclear why these are here 🤔

**Review by liamsi — APPROVED** [review]
Looks good. One minor nit

**Review by SweeXordious — APPROVED** [review]

File: x/payment/types/square_sizes.go (hunk 1 above)
Comment (evan-forbes) [reply x/payment/types/square_sizes.go:8]: whoops, yeah this code was originally just for test code, but those issues describe signing over all applicable square sizes
#236 #239 

This still needs a refactor and we should perform this calculation inside of `NewWirePayForMessage`.

Will do in a follow up PR later today

**Pull request merged**